source/config.js

```diff
@@ -235,6 +239,25 @@ if (typeof currentRootPath !== 'string') {
 }
 module.exports.rootPath = currentRootPath;
 
+let currentGitCommand = module.exports.gitCommand;
+if (typeof currentGitCommand !== 'string') {
+  currentGitCommand = '';
+} else if (currentGitCommand !== '') {
+  // must start with a slash
+  if (currentGitCommand.charAt(0) !== '/') {
+    currentGitCommand = '/' + currentGitCommand;
+  }
+  // can not end with a trailing slash - it should be a command
+  if (currentGitCommand.charAt(currentGitCommand.length - 1) === '/') {
+    currentGitCommand = currentGitCommand.substring(0, currentGitCommand.length - 1);
+  }
+}
+// finally set default if empty
+if (currentGitCommand === '') {
```

Comment:
This will never be hit right due to if claus at the line 247 above?  if null or blank is passed in, it would be '/' and this will never be hit right?

Also, despite the documentation, I'm not sure where window's pathing is being accepted.  Your example will result in `/D:\\Apps\\Git\\bin\\git` right?  Am I missing somehting?